CalibMuon/DTCalibration/interface/DTCalibMuonSelection.h

```diff
@@ -4,26 +4,26 @@
 
 // user include files
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDFilter.h"
+#include "FWCore/Framework/interface/stream/EDFilter.h"
 #include "FWCore/Utilities/interface/EDGetToken.h"
 
 #include "DataFormats/MuonReco/interface/MuonFwd.h"
 
 #include <vector>
 #include <string>
 
-class DTCalibMuonSelection : public edm::EDFilter {
+class DTCalibMuonSelection : public edm::stream::EDFilter<> {
 public:
   explicit DTCalibMuonSelection(const edm::ParameterSet&);
 
   ~DTCalibMuonSelection() override;
 
 private:
-  void beginJob() override;
+  void beginStream(edm::StreamID) override;
```

Comment:
I dont think these were supposed to be changed

Reply:
> think these were supposed to stay beginJob() and endJob()

@tvami There's no provision for such methods in stream modules, cf: https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkStreamModuleInterface#Base_Class so what @yuanchao did is redundant, but OK.
As a matter of fact the methods could be removed as they are not implemented.